src/main/java/org/itech/fhirhose/etl/service/impl/ETLServiceImpl.java

```diff
@@ -221,6 +238,39 @@ private void putPractitionerValuesIntoETLRecord(ETLRecord etlRecord, Practitione
 //					+ fhirPractitioner.getNameFirstRep().getFamily());
 //		}
 	}
+	
+    private void putQuestionnaireResponseValuesIntoETLRecord(ETLRecord etlRecord,
+            QuestionnaireResponse fhirQuestionnaireResponse) {
+        log.trace("putQuestionnaireResponseValuesIntoETLRecord");
+        if (fhirQuestionnaireResponse.hasItem()) {
+            for (QuestionnaireResponseItemComponent item : fhirQuestionnaireResponse.getItem()) {
+                switch (item.getText()) {
+                case "Reason for Visit":
+                    break;
+                case "Countries Vistied within 6 Months":
+                    etlRecord.setCountries_visited(item.getAnswerFirstRep().toString());
```

Comment:
@rossumg  , this element contains more than one ItemAnswerComponent since we get a List of countries.
i think we should just loop through all the available List<QuestionnaireResponseItemAnswerComponent>
with item.getAnswer();